src/fparser/common/tests/test_readfortran.py

```diff
@@ -988,3 +988,56 @@ def test_extract_construct_name():
     name, text_result = extract_construct_name(text_input)
     assert name == "name"
     assert text_result == "stuff"
+
+
+@pytest.mark.parametrize('input_text, ref', [
```

Comment:
Nice use of parametrize

Reply:
All credit goes to @arporter who suggested this :)